libs/utils/src/types.ts

```diff
@@ -97,26 +97,27 @@ export interface Storage {
   clear(): Promise<void>;
 }
 
-export interface CardAbsentApi {
-  present: false;
+export interface CardApiNotReady {
+  status: 'no_card' | 'error';
```

Comment:
suggest something slightly more descriptive than `error`, something specific to the card (as opposed to, say, the reader itself, which we might want to add later.) Something like `error_card_unreadable` or something like it.

Reply:
I was thinking we could add an `errorType` field if we wanted to surface multiple errors later.

Reply:
I can live with that!